hoomd/md/pytest/test_thermoHMA.py

```diff
@@ -0,0 +1,32 @@
+import hoomd
```

Comment:
Let's add tests for logging see `hoomd/md/pytest/test_potential.py` and `hoomd/confest.py` for the relevant tools for this.